pages/[id]/index.js

```diff
@@ -32,6 +32,7 @@ const MemberProfile = ({ imageLink, user, contributions, tasks, errorMessage })
         devUser={devUser}
         tasks={tasks}
       />
+        <br/><br/>
     </Layout>
```

Comment:
How is br / serving the purpose here

Reply:
The <br/> tag creates a line break after the content that has to be displayed before the footer.
Footer will always be displayed at the bottom because that's how it is structured ( in footer.sccss).
Footer is a part of <Layout/> and before footer is displayed at the bottom of the screen, I added these <br/> tags which created a line break after the member contents (such as, active tasks, contribution) got over, which naturally created a margin for the footer

Reply:
Please avoid doing that 😅  While it is a quick hack, it _might_ cause styling issues later on (in case our design guidelines change).
Do checkout [css-tricks.com](https://css-tricks.com/)